docs/packages.md

```diff
@@ -73,38 +74,89 @@ should also be set up with signing keys for packages and your signing
 key should have a passphrase, which we call `<passphrase>` throughout.
 
 All official LinuxKit packages are multi-arch manifests and most of
-them are available for `amd64`, `arm64`, and `s390x`. Official images
-*must* be build on both architectures and they must be build *in
-sequence*, i.e., they can't be build in parallel.
+them are available for the following platforms:
 
-To build a package on an architecture:
+* `linux/amd64`
+* `linux/arm64`
+* `linux/s390x`
 
-```
-DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE="<passphrase>" linuxkit pkg push «path-to-package»
-```
+Official images *must* be built on all architectures. They can be built and
+pushed in parallel, but the manifest *must* be pushed on one platform only, when
+all of the images are done.
```

Comment:
Why? This makes the package push more complicated. By default a push should create the manifest list if not present or update the existing one, and then sign the new manifest list

Reply:
Is that how we want to do it? Part of the issue we had was pushing in parallel could create manifest indexes (technically, indices) that would trounce each other. By saying it this way, we encourage people to build and push in parallel, and then do the manifest at the end.